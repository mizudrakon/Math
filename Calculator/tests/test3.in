5
((((-(((3)))))))
(2-1)*((3-2)*((4-3)*(5.0-4.0)/(6-5)))
[1+3*(7-8+{2})]
(1+(2+(3))
1+2)+3

