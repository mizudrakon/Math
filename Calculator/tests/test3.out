-3
1
4
invalid
invalid
